src/shared_modules/indexer_connector/include/indexerConnector.hpp

```diff
@@ -54,16 +54,12 @@ class EXPORTED IndexerConnector final
     std::unordered_map<std::string, std::chrono::system_clock::time_point> m_lastSync;
 
     /**
-     * @brief Intialize method used to load template data and initialize the index.
+     * @brief Intialize method used to initialize the index.
      *
-     * @param templateData Template data.
-     * @param indexName Index name.
      * @param selector Server selector.
      * @param secureCommunication Secure communication.
      */
-    void initialize(const nlohmann::json& templateData,
-                    const std::shared_ptr<ServerSelector>& selector,
-                    const SecureCommunication& secureCommunication);
+    void initialize(const std::shared_ptr<ServerSelector>& selector, const SecureCommunication& secureCommunication);
```

Comment:
having this in the interface doesn't make sense.

Reply:
Yeap, this was called by the removed thread, removed.